webpack.config.js

```diff
@@ -0,0 +1,30 @@
+var path = require('path');
+
+module.exports = {
+    entry: path.join(__dirname, 'srcjs', 'react-tools.js'),
```

Comment:
Should we add `mode: 'development',` since `react-tools.js` is so small?  Or do you plan to `import` lots of code?  I prefer to have a non-minified file since some authors might not be as familiar with debugging using a sourcemap.

Reply:
@timelyportfolio makes sense; and I have no plans to import code. I pushed an edit to `webpack.config.js` and built new `inst/` JS.